.github/workflows/build-adoptium.yml

```diff
@@ -0,0 +1,25 @@
+name: Build Adoptium
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
```

Comment:
You could do with adding names to these job elements to prevent them just appearing as build in the UI